### PR TITLE
Fix benchmark builds on non-Apple platforms

### PR DESCRIPTION
**Pull request by wilhuff**

### CODE REVIEW

File: cmake/cc_rules.cmake [hunk 1]
```diff
@@ -93,13 +94,15 @@ endfunction()
 #   target
 #   SOURCES sources...
 #   DEPENDS libraries...
+#   [EXCLUDE_FROM_ALL]
```
Comment (rsgowman) [inline cmake/cc_rules.cmake]: This seems unrelated?

(No objection to it though.)

Comment (wilhuff) [reply cmake/cc_rules.cmake]: It was pass one at making this happen. Turned out to not be enough but didn't seem harmful.
